### DISCUSSION TITLE
droplet colours

### DISCUSSION THREAD

**Discussion by kidnamednuclearwarin72hours** [niels747/2D-Weather-Sandbox#68] [Q&A] [answered]
what do each of the droplet colours mean?

**Comment by Meso22** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/105577122/219309754-ace97ead-7869-43b0-a338-eff91aab4fb1.png)

**Reply by kidnamednuclearwarin72hours** [reply]
thanks
